### PR TITLE
test install-scripts

### PR DESCRIPTION
**Pull request by YuliiaKovalova**

### CODE REVIEW

**Comment by dougbu** [comment]
@YuliiaKovalova this PR changes a file that is not actually under our control. We get all eng/common/ files from https://github.com/dotnet/arcade and can't take manual changes here.

That said, is this literally a test rather than something you intend for us to merge❔

File: eng/common/tools.ps1 [hunk 1]
```diff
@@ -263,7 +263,7 @@ function GetDotNetInstallScript([string] $dotnetRoot) {
   if (!(Test-Path $installScript)) {
     Create-Directory $dotnetRoot
     $ProgressPreference = 'SilentlyContinue' # Don't display the console progress UI - it's a huge perf hit
-    $uri = "https://dotnet.microsoft.com/download/dotnet/scripts/$dotnetInstallScriptVersion/dotnet-install.ps1"
+    $uri = "https://raw.githubusercontent.com/dotnet/install-scripts/main/src/dotnet-install.ps1"
```
[the remark this answers is not in the document]
Comment (YuliiaKovalova) [reply eng/common/tools.ps1:266]: It's a draft PR for test purposes only.
This approach is used for triggering build/test run actions in the pipeline.
Sorry for getting you confused.

**Comment by japui** [comment]
scripts de instalação de teste #

**Comment by dougbu** [comment]
@YuliiaKovalova should we close this PR❔ It's not something we can accept because the next Arcade update would undo any changes. However, I don't see any related changes in the current Arcade repo nor an open PR from you.

**Pull request closed without merging**